AUTHORS

```diff
@@ -6,4 +6,5 @@
 # Name or Organization <email address>
 # The email address is not required for organizations.
 
-Google Inc.
\ No newline at end of file
+Google Inc.
+Anand Suresh
```

Comment:
I don't believe contributors are added to this list. Can @jgeewax comment on this?